resources/chat-types.js

```diff
@@ -1,18 +1,20 @@
 module.exports = {
+  'OTHER': 0,
```

Comment:
use `-1` so it is clearer?

Reply:
@jayvdb I was thinking that it is probably used as index of an array on some implementation, and if it passed as -1 it may get the last element of the array.